### PR TITLE
chore: add embark colorscheme and update readme

### PR DESCRIPTION
**Pull request by hlidotbe**
Hi @skbolton ,

Here's the PR for Alacritty theme!

### CODE REVIEW

**Review by skbolton — CHANGES_REQUESTED** [review]
Can you also include a screenshot in the readme by chance?

**Comment by hlidotbe** [comment]
Sorry I lost track of this request. I've added a screenshot as requested.

**Comment by skbolton** [comment]
So a few colors in the palette have changed for terminals. Leads to better visual selection and whatnot. If you are ready to be done with this I can get this merged on work on updating it. If you are up to it then the [kitty](https://github.com/embark-theme/kitty/blob/master/embark-theme.conf) repo shows the new colors. Let me know what you think about this.

**Comment by hlidotbe** [comment]
I've updated the colors based on the Kitty theme. They should match now (updated the screenshot as well)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
 # alacritty
 An ambitious theme for alacritty
+
+![Screenshot](./screenshot.png)
+
+## How to use
+
+Edit alacritty configuration file (usually located at `$XDG_CONFIG_HOME/kitty/kitty.conf`) and copy embark to your
```
Comment (skbolton) [inline README.md]: Just need to update this path for alacritty

**Review by skbolton — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hlidotbe) [reply README.md]: :facepalm: done

**Comment by skbolton** [comment]
BTW there is a bashtop if you want to check it out.

**Comment by skbolton** [comment]
Also added a link to this from the main site - https://embark-theme.github.io/
I think the rocket is a good logo for alacritty wdyt?

**Comment by hlidotbe** [comment]
Yes, the rocket should fit although it's still a terminal like kitty or iTerm so idk. Vim has its logo, bashtop, kitty and iterm are more a "concept". Honestly I think it's fine either way

**Pull request merged**